ext/standard/tests/mail/gh7875.phpt

```diff
@@ -0,0 +1,35 @@
+--TEST--
+GH-7875 (mails are sent even if failure to log throws exception)
+--INI--
+sendmail_path={MAIL:{PWD}/gh7875.mail.out}
+mail.log={PWD}/gh7875.mail.log
+--FILE--
+<?php
+function exception_error_handler($severity, $message, $file, $line) {
+    if (!(error_reporting() & $severity)) {
+        return;
+    }
+    throw new ErrorException($message, 0, $severity, $file, $line);
+}
+set_error_handler("exception_error_handler");
+
+touch(__DIR__ . "/gh7875.mail.log");
+chmod(__DIR__ . "/gh7875.mail.log", 0444);
+
+try {
+	$return = mail('recipient@example.com', 'Subject', 'Body', []);
+	var_dump($return);
```

Comment:
```suggestion
	mail('recipient@example.com', 'Subject', 'Body', []);
	echo 'Not Reached';
```
Because the warning will be converted into an exception, `mail()` will never return and thus the `var_dump()` is never going to be reached.

I'd suggest to update the test to make this clear and to prevent the output of this `var_dump()` from being confused by the one in line 24 in case the exception does not work properly for whatever reason.

Reply:
That makes sense. Thank you!